src/utils/customHooks.js

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react';
+
+export const usePosition = () => {
```

Comment:
wouldn't be `src/hooks/usePosition.js` better place for it?

Reply:
Yes, this would be better